src/renderer/components/+cluster-settings/settings-group-status.tsx

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import { Spinner } from "../spinner";
+import { Cluster } from "../../../main/cluster";
+
+interface Props {
+  cluster: Cluster;
+}
+
+export class Status extends React.Component<Props> {
+  renderStatusRows(): JSX.Element[] {
+    const { cluster } = this.props;
+    
+    const rows = [
+      <h5 key="online-header">Online Status</h5>,
+      <span key="online-value">{cluster.online ? "online" : `offline (${cluster.failureReason || "unknown reason"})`}</span>,
+      <h5 key="distribution-header">Distribution</h5>,
+      <span key="distribution-value">{cluster.distribution}</span>,
+      <h5 key="version-header">Kernel Version</h5>,
+      <span key="version-value">{cluster.version}</span>,
+      <h5 key="api-header">API Address</h5>,
+      <span key="api-value">{cluster.apiUrl}</span>,
+    ];
+
+    if (cluster.nodes > 0) {
+      rows.push(
+        <h5 key="count-header">Nodes Count</h5>,
+        <span key="cound-value">{cluster.nodes}</span>
+      );
+    }
+
+    return rows;
+  }
+
+  render() {
+    const { cluster } = this.props;
+
+    return <div className="ClusterSettingsGrouping">
+      <h2>Status</h2>
+      <hr/>
+      <h4>Cluster status</h4>
+      <p>
+        Cluster status information including: detected distribution, kernel version, and online status.
+      </p>
+      {
+        cluster.initialized
```

Comment:
Most probably you will never see `cluster.initialized == false` at this stage, so the spinner will never show.

Reply:
Fair, when I first starting this work I was seeing `cluster.initialized` every time I did a `⌘+R` (reload). Will remove.